app/assets/fonts/Canterbury/Canterbury.svg

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
I don't know how to go about doing this. I assume keep .svg as is locally, but do some sort of special thing with heroku to put fonts on S3...?